### ISSUE TITLE
Profile Settings should stop keybinds from working.

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#206]
When in Profile Settings you have complete access to the keyboard shortcuts and can navigate and delete cards/open/close them.

To delete a card select a card and click on your profile then Profile Settings.
Press the Backspace key 2 times.
The card is removed.

https://user-images.githubusercontent.com/65631811/199460849-29a1cf12-15d5-4489-86da-42bdd70ebf5f.mp4

**Comment by Connoropolous** [comment]
did we fix this one?

**Comment by TechSupport26** [comment]
No. Not yet. I can still navigate while the Profile Settings UI is visible.
